### DISCUSSION TITLE
SSL/TLS

### DISCUSSION THREAD

**Discussion by gdg316** [tsightler/ring-mqtt#466] [General Discussion]
How would I set up SSL/TLS for Mosquito Broker & Ring MQTT once I get everything communicating correctly. I know files have to be in SSL folder. 
Any help or instructions would be helpful.

**Comment by tsightler** [comment]
These discussions groups are not for general questions regarding MQTT setup.  There are already plenty of resources and forums available to ask general question and nearly everything that could ever be asked about MQTT setup on Home Assistant already has a thread on the Home Assistant forums.  Please use Google and other search tools and, if you can't figure it out, ask for assistance on the Home Assistant forums.
